pages/docs/tutorials/validate-documents.md

```diff
@@ -0,0 +1,126 @@
+Let's pretend we have an invalid AsyncAPI document.
+
+1. Open [Studio](https://studio.asyncapi.com/).
+
+2. Copy and paste the below invalid AsyncAPI document:
+
+```yaml
+{`asyncapi: '1.0.0'
+info:
+  title: Streetlights API
+  version: '1.0.0'
+  license:
+    name: Apache 2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0'
+servers:
+  mosquitto:
+    url: mqtt://test.mosquitto.org
+    protocol: mqtt
+channels:
+  light/measured:
+    publish:
+      summary: Inform about environmental lighting conditions for a particular streetlight.
+      operationId: onLightMeasured
+      message:
+        name: LightMeasured
+        payload:
+          type: object
+          properties:
+            id:
+              type: integer
+              minimum: true
+              description: Id of the streetlight.
+            lumens:
+              type: integer
+              minimum: 0
+              description: Light intensity measured in lumens.
+            sentAt:
+              type: integer
+              format: date-time
+              description: Date and time when the message was sent.`}
+ ``` 
+## Troubleshoot Studio errors
+Let's fix the errors one by one until we end up with a valid AsyncAPI document.
+ 
+1. You can see the error message on the screen: `Empty or invalid document. Please fix errors/define AsyncAPI document.`
+
+2. Open diagnostics, you can see more information related to your errors.
+
+3. Fix the incorrect AsyncAPI specification number to `2.5.0`.
+ 
+```yaml
+asyncapi: '2.5.0'
+info:
+  title: Account Service
+  version: 1.0.0
+  ```
+<Remember>
+Notice how <b>description</b> property is missing; that doesn't make the AsyncAPI document invalid, but it's always better to include.
+</Remember>
+
+4. Read the next error: `must be number`. Fix the `minimum` by changing it to: `0`.
+
+```yaml
+          properties:
+            id:
+              type: integer
+              minimum: 0
+``` 
+5. You see three errors:
+- must be equal to one of the allowed values
+- must be array
+- must match a schema in `anyOf`
+
+`anyOf` means it should match any one the above schemas then it is valid.
+
+Now let's fix this error by changing the type to `string`
+
+```yaml
+            sentAt:
+              type: string
+              format: date-time
+              description: Date and time when the message was sent.`}
+```
+
+6. Congratulations! You identified and fixed all the errors, and now have a valid AsyncAPI document.
+
+## Summary
+This tutorial taught us how to validate an AsyncAPI document using the AsyncAPI Studio tool. We also learned to troubleshoot an invalid AsyncAPI document by following the error message directions in diagnostics. In doing so, we learned how to identify `REQUIRED` properties in all AsyncAPI documents.
+
+## Next steps
+Now that you've completed this tutorial, go ahead to learn how to [validate messages/events]() which you will be sending to your application.
```

Comment:
next step do not rever to the same doc as the buttons below, which is strange

also look at https://deploy-preview-1022--asyncapi-website.netlify.app/docs/tutorials/validate-documents#now-that-youve-completed-this-tutorial-go-ahead-to-learn-how-to-validate-messagesevents-which-you-will-be-sending-to-your-application

strange stuff with section, and to be honest I don't  know why 🤔 
try maybe `---` must have an empty line before

Reply:
fixed